### PR TITLE
File class

### PR DESCRIPTION
**Pull request by jaymeans**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import { EventEmitter } from 'eventemitter3'
-import Promise from 'ts-promise'
```
Comment (dboskovic) [inline src/index.ts]: Let's verify this still works in a non-promise browser. Seems like TS is building a promise shim in, but want to make sure.

File: src/index.ts [hunk 2]
```diff
@@ -19,19 +17,20 @@ export default class FlatfileImporter extends EventEmitter {
...
 
   private $resolver: (data: any) => any
   private $rejecter: (err: any) => any
 
-  constructor (apiKey: string, options: object) {
+  constructor (apiKey: string, options: object, customer: object = {}) {
     super()
     this.apiKey = apiKey
     this.options = options
+    this.customer = customer
```
Comment (dboskovic) [inline src/index.ts]: Let's not instantiate with customer and let's add a `setCustomer` function

File: src/index.ts [hunk 3]
```diff
@@ -96,6 +96,35 @@ export default class FlatfileImporter extends EventEmitter {
     })
   }
 
+  /**
+   * Use loadMeta() when you want a promise returned. This is necessary if you want to use
+   * async/await for an es6 implementation
+   */
+  loadMeta (): Promise<FlatfileResults> {
```
Comment (dboskovic) [inline src/index.ts]: Let's call this `loadData()`

File: src/index.ts (hunk 2 above)
Comment (jaymeans) [reply src/index.ts]: There's an updateCustomer function, should we change it to setCustomer, or add a new setCustomer function?

File: src/index.ts (hunk 1 above)
Comment (jaymeans) [reply src/index.ts]: I don't have a local IE to test it on. Suggestions?

**Pull request merged**